src/components/combobox/combobox.tsx

```diff
@@ -281,6 +281,7 @@ export class Combobox
   }>;
 
   /** Called when a selected item in the combobox is dismissed via its chip */
+  // eslint-disable-next-line @esri/calcite-components/require-event-emitter-type
```

Comment:
https://github.com/Esri/calcite-components/pull/5001 will take care of these. Sorry!

Reply:
Should we hold off on this change until its ready to go?